src/index.d.ts

```diff
@@ -0,0 +1,26 @@
+declare module "dwolla-v2" {
+  export class Client {
+    get: (
+      path: string,
+      query?: { [prop: string]: string },
+      headers?: { "Idempotency-Key"?: string } & { [prop: string]: string }
+    ) => Promise<Response>;
+    post: (
+      path: string,
+      body?: object,
+      headers?: { "Idempotency-Key"?: string } & { [prop: string]: string }
+    ) => Promise<Response>;
+    delete: (path: string) => Promise<Response>;
+    constructor(options: {
+      key: string;
+      secret: string;
+      environment?: "production" | "sandbox";
+    });
+  }
+
+  interface Response {
+    status: number;
+    headers: Headers;
```

Comment:
This doesn't match the headers that node-fetch has:
https://github.com/Dwolla/dwolla-v2-node/blob/main/src/dwolla/Token.js#L1

https://github.com/node-fetch/node-fetch/blob/master/src/headers.js#L42

You can see that node-fetch supports `raw()` for example. Is this on purpose?

Reply:
Yeah, I don't see us exposing the types of the underlying HTTP library unless there's a good reason to. Keeps things more flexible in the future if we were to swap out HTTP libraries or something.

Open to thoughts.